stable/appmesh-inject/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 name: appmesh-inject
 description: App Mesh Inject Helm chart for Kubernetes
-version: 0.12.0
-appVersion: 0.4.0
+version: 0.12.1
```

Comment:
we need to bump to 0.13.0 to align with (https://github.com/aws/eks-charts/pull/107)